cmd/config.go

```diff
@@ -0,0 +1,57 @@
+package cmd
+
+import (
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+	"github.com/zricethezav/gitleaks/v8/manage"
+	"os"
+	"path/filepath"
+)
+
+func init() {
+	rootCmd.AddCommand(configCmd)
+	configCmd.AddCommand(configFetchCmd)
```

Comment:
Thoughts on adding [gitleaks config reset](https://github.com/gitleaks/gitleaks/issues/1557#issuecomment-2408765315)?

All it would do is delete the config file if it exists:

With output looking something like:
```
$ gitleaks config reset
Config file at /home/username/.config/gitleaks/config.toml removed

$ gitleaks config reset
No config file at /home/username/.config/gitleaks/config.toml found
Nothing to do
```
or something similar.


Rationale:

- Helps a user reset their config if it gets wonky (gitleaks version upgrade borks something, etc)
- Keeps them from needing to remember where their config file is or even if they have one. They can just do this to make sure an environment is reset back to default